### PR TITLE
Add `extra` field to Proposal model

### PR DESCRIPTION
**Pull request by FelipeRosa**

### CODE REVIEW

File: src/vit-servicing-station/vit-servicing-station-lib/migrations/postgres/2022-11-29-134506_add_proposals_extra_fields/up.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
+ALTER TABLE proposals ADD proposal_extra_fields TEXT;
```
Comment (FelipeRosa) [inline src/vit-servicing-station/vit-servicing-station-lib/migrations/postgres/2022-11-29-134506_add_proposals_extra_fields/up.sql:1]: Using `TEXT` here so vit-ss code remains compatible with Postgres and SQLite (I guess SQLite supports JSON type only as an extension).

**Comment by stevenj** [comment]
I am happy with this.  Aligns with a change I had already made in the new schema.

**Review by stevenj — APPROVED** [review]
LGTM

**Comment by abhi-iog** [comment]
Thank you @FelipeRosa !

**Review by saibatizoku — APPROVED** [review]

**Pull request merged**